lib/src/rules/dartdoc_summary_line.dart

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2016, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// ignore_for_file: slash_for_doc_comments
+
+import 'dart:convert';
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+
+import '../analyzer.dart';
+
+/// The maximum allowed length for a summary line.
+const maxSummaryLength = 140;
```

Comment:
I don't think we want to introduce a max like this. It is not backed by the style guide, and I think will introduce many false negatives, which reduce the usefulness of the lint rule.

Reply:
The style guide says "brief" but doesn't specify how many characters is "brief" :P

Reply:
I turned this into "the number of lines that make the comment". So if for whatever reason you need to use a line break (say your line is 80+ lines long, but you have a strict enforcement of 80 columns), it'll let you use up to 2 lines. I think that should help with "brevity" without being too specific about the number of chars.